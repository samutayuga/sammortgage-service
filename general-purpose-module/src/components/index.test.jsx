import {describe,expect, it} from 'vitest'
import renderer from 'react-test-renderer'
import Greeting from './index.jsx';
import React from 'react'

describe('Greeting Component',()=>{
    it('Greeting component renders correctly',()=>{
        const component = renderer.create(
            <Greeting/>
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
    it('prop working',()=>{
        const component = renderer.create(
            <Greeting message={'Universe'}/>
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
}
)