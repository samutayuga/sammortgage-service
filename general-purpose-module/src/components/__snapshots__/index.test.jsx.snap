// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Greeting Component > Greeting component renders correctly 1`] = `
<div>
  Hello 
</div>
`;

exports[`Greeting Component > prop working 1`] = `
<div>
  Hello 
  Universe
</div>
`;
